### DISCUSSION TITLE
How do you make rich presence for a game that isn't yours?

### DISCUSSION THREAD

**Discussion by RubyPiec** [zmxhawrhbg/gdrpc#6] [Q&A]
This is extremely unrelated but I really want to know, this will be really nice to me.

**Comment by qimiko** [comment]
hi
my initial version of the rich presence had an overuse of pointers, if you're just starting out in terms of game modification i would likely start by finding relevant pointers that describe things like the state of the player and other variables
if you want to go further then you can also find addresses to functions that control the pointers so you don't have to constantly scan for changes and can instead make a nice hook
from there it's just a matter of hooking up these changes to update a presence and whatnot

**Comment by RubyPiec** [comment]
woah, thank you! I think I'm gonna wait until the day there's a tutorial with something like this.
